.github/workflows/test-devcontainer.yaml

```diff
@@ -0,0 +1,65 @@
+name: Test DevContainer Image
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test-devcontainer:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Setup Go 1.22.x
+        uses: actions/setup-go@v5
+        with:
+          go-version: "1.22.x"
+
+      # required for installing devcontainer CLI
+      - name: Setup NodeJS 20.x
+        uses: actions/setup-node@v4
+        with:
+          node-version: "20.x"
+
+      - name: Setup Devcontainer CLI
+        run: |
+          npm install -g @devcontainers/cli
+
+      - name: Generate sample Kubebuilder project
+        run: |
+          make install
+          mkdir test-project
+          cd test-project
+          ../bin/kubebuilder init --domain tutorial.kubebuilder.io --repo tutorial.kubebuilder.io/project
```

Comment:
It is fine create the project.
But we also have the samples already
So here we could only
cd testdata/project-v4 
Then, just run the tests

Reply:
Didn't notice this! Will refactor it.